avocado/utils/disk.py

```diff
@@ -9,18 +9,84 @@
 #
 # See LICENSE for more details.
 #
-# This code was inspired in the autotest project,
+# Copyright: 2017 IBM
+# Author: Narasimhan V sim@linux.vnet.ibm.com
 #
+# freespace() function was inspired in the autotest project,
 # client/base_utils.py
 
 
 """
 Disk utilities
 """
 
+import logging
 import os
+from . import process
+
+
+LOGGER = logging.getLogger('avocado.test')
+
+
+class FSException(Exception):
+    """
+    Base Exception Class for all exceptions
+    """
+    pass
+
+
+class DiskException(Exception):
+    """
+    Base Exception Class for all exceptions
+    """
 
 
 def freespace(path):
+    """
+    Gets freespace on the given path.
+    """
     fs_stats = os.statvfs(path)
     return fs_stats.f_bsize * fs_stats.f_bavail
+
+
+def zero_disk(disk_name):
```

Comment:
Not really. I wanted mainly to clear the FS headers, and thought of this way to do it.
And planned to extend functionalities in future.